Vagrantfile

```diff
@@ -0,0 +1,28 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  # All Vagrant configuration is done here. The most common configuration
+  # options are documented and commented below. For a complete reference,
+  # please see the online documentation at vagrantup.com.
+
+  # Every Vagrant virtual environment requires a box to build off of.
+  config.vm.box = "phusion/ubuntu-14.04-amd64"
```

Comment:
```
(master) ~/test ~> vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Box 'phusion/ubuntu-14.04-amd64' could not be found. Attempting to find and install...
[etc]
```

Cool :-) before you had to previously download the boxes if they were not on the system. Ok ok, I should use vagrant more often. 

Tack igen!